main/lxterminal/APKBUILD

```diff
@@ -1,33 +1,31 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=lxterminal
-pkgver=0.3.0
-pkgrel=2
+pkgver=0.3.1
+pkgrel=0
 pkgdesc="Desktop-independent VTE-based terminal emulator"
 url="http://lxde.org/"
 arch="all"
 license="GPL-2.0-or-later"
-depends=""
-makedepends="vte-dev"
+depends="vte3"
+makedepends="autoconf automake docbook2x intltool vte3-dev"
 subpackages="$pkgname-doc $pkgname-lang"
-source="https://downloads.sourceforge.net/lxde/$pkgname-$pkgver.tar.xz
-	CVE-2016-10369.patch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/lxde/lxterminal/archive/$pkgver.tar.gz"
 builddir="$srcdir"/$pkgname-$pkgver
 
-# secfixes:
-#   0.3.0-r1:
-#     - CVE-2016-10369
```

Comment:
Please do not remove the secfixes comments. They're there to indicate the historical precedence of CVEs for the package.